karma.conf.js

```diff
@@ -1,51 +1,43 @@
+const webpackConfig = require("webpack-config-narazaka-ts-js").webTest;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->